net.puddletag.puddletag.appdata.xml

```diff
@@ -0,0 +1,66 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
This file should be submitted and maintained upstream.

Reply:
As for the patch, the developer does not want to have the file, that's the only reason I included it here.